examples/getting-started/components/Posts.tsx

```diff
@@ -37,32 +33,35 @@ function Posts({
         )}
         {intro && <p className={styles.intro}>{intro}</p>}
         <div className="posts">
-          {thePosts &&
-            thePosts.map((post) => (
+          {posts &&
+            posts.map((post) => (
               <div
                 className={styles.single}
                 key={post.id}
                 id={`post-${post.id}`}>
                 <div>
                   <Heading level={postTitleLevel} className={styles.title}>
                     <Link href={post.uri}>
-                      <a href={post.uri}>{post.title}</a>
+                      <a>{post.title}</a>
                     </Link>
                   </Heading>
                   <div
                     className={styles.excerpt}
                     // eslint-disable-next-line react/no-danger
                     dangerouslySetInnerHTML={{ __html: post.excerpt ?? '' }}
                   />
-                  <a
-                    href={post.uri}
-                    aria-label={`Read more about ${post.title}`}>
-                    {readMoreText}
-                  </a>
+                  <Link href={post.uri}>
+                    <a
+                      aria-label={`Read more about ${
+                        post.title || ' the post'
```

Comment:
```suggestion
                        post.title || 'the post'
```

^ Removes extra space before fallback text, since there's already a space after "Read more about".

Reply:
Well spotted, thanks!